specs.tex

```diff
@@ -66,6 +66,11 @@ \subsection{Arena}
         that of other zones.
   \item Tokens will be placed in the scoring zone on the opposite side to their
         matching coloured scoring zone.
+  \item \label{spec:flags} Flags will be made from wire folded in half
```

Comment:
* What restrictions are there about the flag and how the flag is mounted?
* Could the pole be horizontal?
* Can the flag be an April Tag Marker?
* Does the flag need to be at some minimum (or maximum) height?
* Does the flag need to be within the 50cm bounding box of the robot at the start of the match?